book/README.md

```diff
@@ -15,7 +15,6 @@ The book is continuously rendered [here](https://paradigmxyz.github.io/reth/)!
 
 **[Getting Started]()**
 To get started with Reth, install, configure and sync your node.
-* To install and build reth, you can use the following [installation instruction](./installation.md).
-
+* To install and build reth, you can use the following [installation instruction](./getting_started/README.md).
```

Comment:
Looks like you removed the old file (`./installation.md`), could you re-add it as `./getting_started/README.md`? You wrote some great instructions in there!

Reply:
Yes, I remove it by mistake.